clients/drcachesim/tests/burst_syscall_inject.cpp

```diff
@@ -102,24 +103,13 @@ write_instr_entry(void *dr_context, std::unique_ptr<std::ostream> &writer, instr
     }
     instr_encode_to_copy(dr_context, instr, encoding.encoding, instr_app_pc);
     write_trace_entry(writer, encoding);
-    write_trace_entry(
-        writer,
-        make_instr(reinterpret_cast<addr_t>(instr_app_pc), TRACE_TYPE_INSTR, len));
+    write_trace_entry(writer,
+                      make_instr(reinterpret_cast<addr_t>(instr_app_pc), type, len));
 }
 
-static std::string
-write_system_call_template(void *dr_context)
+void
```

Comment:
This lost its static decl.  I guess in C++ if in an anon namespace it's not needed: but I don't see an anon namespace in this file.

The write_footer_entries below could also be static.

Reply:
Fixed. Also for various others in this file.